haskell-completions.el

```diff
@@ -0,0 +1,189 @@
+;;; haskell-completions.el --- Haskell Completion package
+
+;; FIXME copyright (c) 2015 Haskell mode community. All rights reserved.
+
+;; Authors:
```

Comment:
Originally, I've copied this from existing package, and replaced sigular `Author` to plural `Authors`. Not sure, is it right?